code/style.css

```diff
@@ -1,4 +1,451 @@
+
+}
+
+.bigArticle .dates {
+  margin: 30px 30px;
+}
+
+.big-news img {
+  width:100%;
+  max-height: 700px;
+  object-fit: cover;
+}
+
+
+/** Small News **/
+
+
+.small-news {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-evenly;
+  animation-name: move;
+  animation-duration: 2s;
+  animation-timing-function: ease-in;
+  animation-fill-mode: backwards;
+  animation-delay: 2s;
+}
+
+.smallArticle {
+  margin: 20px 0px;
+  width: 90%;
+  border-top: 2px solid  rgb(152, 192, 164);
+  box-shadow: 10px 10px 10px  rgba(152, 192, 164,0.1);
+  background: white;
+}
+
+.smallArticle img {
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+}
+
+h2 {
+  font-size: 20px;
+}
+
+h3 {
+  margin: 10px 20px;
+  font-size: 12px;
+  text-transform: uppercase;
+  font-weight: lighter;
+}
+
+/** Small News - Second Priority */
+
+.small-news-second {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+
+/*  FOOTER  */
+
+footer {
+  background: black;
+  height: 100px;
+  margin-top: 70px;
+  padding: 30px;
+  display: flex;
+  justify-content: center;
+}
+
+.footer-content {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.links-prio-1 {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.links-prio-2 {
+  display: none;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+footer p {
+  color: white;
+  font-size: 13px;
+}
+
+footer a {
+  margin: 0px;
+}
+
+footer a:hover {
+  text-decoration: underline;
+}
+
+/*** PHONE ***/
+
+@media (max-width: 768px) {
+
+  .small-news-second .smallArticle {
+    display: flex;
+    align-items: center;
+    margin: 10px 0px;
+  }
+  
+  .small-news-second .smallArticle h3{
+    display: none;
+  }
+  
+  .small-news-second .smallArticle .articleText{
+    margin: 0px 20px;
+    width: 100%;
+    
+  }
+  
+  .small-news-second .smallArticle h2{
+    font-size: 15px;
+    
+  }
+  
+  .small-news-second .smallArticle p{
+    font-size: 15px;
+    margin: 0px;
+  }
+  
+  .small-news-second .smallArticle .dates{
+    font-size: 10px;
+    margin: 5px;
+  }
+  
+  .small-news-second .smallArticle img {
+    width: 30%;
+    height: 100px;
+    object-fit: cover;
+  }
+  
+  }
+
+/*** TABLET ***/
+
+@media (min-width: 768px) {
+
+  header {
+    justify-content: center;
+    align-items: center;
+  }
+
+  .header-content {
+    justify-content: center;
+    align-items: center;
+  }
+
+  .logo {
+   display: none;
+
+  }
+
+  .hamburger {
+    display:none;
+  }
+
+  nav {
+    display: flex; 
+    width: 100%;
+    justify-content: space-between;
+    
+  }
+
+  .smallArticle {
+    width: 43%; 
+  }
+
+  .links-prio-2 {
+    display: flex;
+  }
+}
+
+/*** DESKTOP ***/
+
+@media (min-width: 1025px) {
+
+/*HEADER*/ 
+
+  header {
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .header-content {
+    width: 65%;
```

Comment:
You should avoid setting this value for `.header-content` - there is no reason for that, let's leave setting its width by its children. If you will check this element id devtools you will see that it's shorter than content (links) inside it